.github/workflows/greetings.yml

```
@@ -0,0 +1,12 @@
+name: Greetings
+
+on: [push]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        message: "Test"
```

actionlint (1.7.4)

input "message" is not defined in action "actions/first-interaction@v1". available inputs are "issue-message", "pr-message", "repo-token"

(action)